migrations/versions/201603311719_da0ddef1744_partially_added_abstract_judgements_to_.py

```diff
@@ -0,0 +1,72 @@
+"""Partially added abstract judgements to Postgres
+
+Revision ID: da0ddef1744
+Revises: 4878972d7e2f
+Create Date: 2016-03-31 17:19:33.252461
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from indico.core.db.sqlalchemy import UTCDateTime
+
+
+# revision identifiers, used by Alembic.
+revision = 'da0ddef1744'
+down_revision = '4878972d7e2f'
+
+
+def upgrade():
+    op.create_table('judgements',
+                    sa.Column('id', sa.Integer(), nullable=False),
+                    sa.Column('creation_dt', UTCDateTime, nullable=False),
+                    sa.Column('abstract_id', sa.Integer(), nullable=False),
+                    sa.Column('track_id', sa.Integer(), nullable=False),
+                    sa.Column('user_id', sa.Integer(), nullable=False),
+                    sa.Column('accepted_type_id', sa.Integer(), nullable=True),
+                    sa.ForeignKeyConstraint(['abstract_id'],
+                                            [u'event_abstracts.abstracts.id'],
+                                            name=op.f('fk_judgements_abstract_id_abstracts')),
+                    sa.ForeignKeyConstraint(['accepted_type_id'],
+                                            [u'events.contribution_types.id'],
+                                            name=op.f('fk_judgements_accepted_type_id_contribution_types')),
+                    sa.ForeignKeyConstraint(['user_id'],
+                                            [u'users.users.id'],
+                                            name=op.f('fk_judgements_user_id_users')),
+                    sa.PrimaryKeyConstraint('id', name=op.f('pk_judgements')),
+                    schema='event_abstracts')
+    op.create_index(op.f('ix_judgements_abstract_id'),
```

Comment:
like in the other revision, `index=True` can be used for pretty much all of those indexes and the constraint names can go away as they are autogenerated